Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -0,0 +1,93 @@
+package com.paypal.android.api.services
+
+import com.google.gson.JsonObject
+import com.paypal.android.api.model.AccessToken
+import com.paypal.android.api.model.CreateOrderRequest
+import com.paypal.android.api.model.Order
+import com.paypal.android.usecase.UpdateOrderUseCase
+import com.paypal.checkout.order.OrderRequest
+import okhttp3.OkHttpClient
+import okhttp3.ResponseBody
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.Body
+import retrofit2.http.PATCH
+import retrofit2.http.POST
+import retrofit2.http.Path
+import java.util.concurrent.TimeUnit
+
+private const val CONNECT_TIMEOUT_IN_SEC = 20L
+private const val READ_TIMEOUT_IN_SEC = 30L
+private const val WRITE_TIMEOUT_IN_SEC = 30L
+
+private val DEFAULT_ACCESS_TOKEN: String? = null
+private val DEFAULT_ORDER_ID: String? = null
```

Comment:
Take it or leave it: adding a comment above these variables to specify what they're for / make it more clear to less experienced devs how to set these.

Ex:
```suggestion
// To hardcode an access token and order ID for this demo app, set the below values:
private val DEFAULT_ACCESS_TOKEN: String? = null // = "your-token"
private val DEFAULT_ORDER_ID: String? = null // = "your-order-id"
```